src/app/common/services/stories.service.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { map, Observable } from 'rxjs';
+import { apiUrl } from '../api-url';
+import { IStory } from '../interfaces/story.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class StoriesService {
+  constructor(private http: HttpClient) {}
+
+  getAllStories(): Observable<IStory[]> {
+    return this.http.get<IStory[]>(apiUrl + 'posts');
+  }
+
+  getStoriesById() {}
```

Comment:
In such cases you can leave `// TODO: Implement`, it helps in future to find such places by simple search on project